agenda-webapps/src/main/resources/locale/portlet/Agenda_en.properties

```diff
@@ -264,3 +264,7 @@ agenda.datePollSwitchedToEvent=Poll changed to event
 agenda.eventSwitchedToDatePoll=Event changed to poll
 agenda.undoRemoveDatePoll=Undo
 agenda.datePollDeleteSuccess=Date poll deleted
+
+#SpaceApplications
+SpaceSettings.application.agenda.application.title=Agenda Application
```

Comment:
@mdiha
Those labels are used by `SpaceSettings`, thus adding them in this file is not sufficient. BTW, correct me if I'm wrong, if I add those FR labels (in Agenda_fr.properties), I don't have the translated labels displayed.
In fact, this fix isn't sufficient, it's a whole improvement to make in API to be able to translate those labels. Either change the Backend to retrieve those labels in [REST Endpoints](https://github.com/Meeds-io/social/blob/develop/component/service/src/main/java/org/exoplatform/social/rest/impl/space/SpaceRestResourcesV1.java#L817) or the frontend to introduce more flexibility. I 'm available to help for this improvement if needed.